docs/build/introduction-to-qasm.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Introduction to OpenQASM
+description: An introduction to OpenQASM (Open quantum assembly language)
+---
+
+# Introduction to OpenQASM
+
+OpenQASM (open quantum assembly language), a machine-independent programming interface for IBM Quantum systems, is an imperative programming language for describing quantum circuits. OpenQASM uses the quantum circuit model to express quantum programs as ordered sequences of parameterized operations (such as gates, measurements, and resets) and real-time classical computation. In addition to quantum algorithms, OpenQASM can describe circuits intended to characterize, validate, or debug quantum systems.
```

Comment:
This turn of phrase sounds like OpenQASM is only for IBM systems: "a machine-independent programming interface for IBM Quantum systems". Or at the very least, it sounds like an oxymoron. Rephrase?

Reply:
How about "a machine-independent programming interface compatible with IBM Quantum Systems"?

Reply:
Yes, that works.